snapcraft/plugins/kernel.py

```diff
@@ -387,6 +387,25 @@ def _parse_kernel_release(self):
                 )
             )
 
+    def _get_kernel_version(self):
+        kernelversion = (
+            subprocess.check_output("make kernelversion", shell=True, cwd=self.builddir)
```

Comment:
what if this fails? How about catching the exception and raising one that inherits from `SnapcraftError`. Look at the `dump` plugin for inspiration